x-pack/plugins/enterprise_search/public/applications/esre/components/layout/page_template.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { ESRE_PLUGIN } from '../../../../../common/constants';
+import { SetEsreChrome } from '../../../shared/kibana_chrome';
+import { EnterpriseSearchPageTemplateWrapper, PageTemplateProps } from '../../../shared/layout';
+import { useEnterpriseSearchNav } from '../../../shared/layout';
+import { SendEnterpriseSearchTelemetry } from '../../../shared/telemetry';
+
+export const EnterpriseSearchEsrePageTemplate: React.FC<PageTemplateProps> = ({
```

Comment:
I was not a 100% sure when to use the `EnterpriseSearch...` prefix in front of components. Let me know if I missed it.